packages/dnb-eufemia/src/fragments/drawer-list/DrawerList.d.ts

```diff
@@ -107,8 +191,18 @@ export type DrawerListOptionsProps = {
 export type DrawerListItemProps = {
   children: React.ReactNode;
   selected: boolean;
+  /**
+   * Define a preselected data entry (index) or key inside an array item. Can be a string or integer.
```

Comment:
```suggestion
   * Define a preselected data entry (index) or key inside an array item. Can be a string or number.
```
This shouldn't be that confusing for a developer, but nice to be consistent